CONTRIBUTING.md

```diff
@@ -1 +1 @@
-See the [contributing guide](https://github.com/pulsar-edit/.github/blob/main/CONTRIBUTING.md)
+See the [Pulsar contributing guide](https://github.com/pulsar-edit/.github/blob/main/CONTRIBUTING.md)
```

Comment:
```suggestion
[See how you can contribute](https://github.com/pulsar-edit/.github/blob/main/CONTRIBUTING.md)
```

Modified the link placement, according to @icecream17 (on Discord) about how a screen reader would handle this text fragment.

Reply:
@Spiker985 or @pulsar-edit/documentation are we okay commiting this modified version, and merging this branch to main?

Reply:
mhm!